### ISSUE TITLE
"Automatically add users with the domain name" feature is not working (autoAddUsersWithDomainName)

### ISSUE THREAD

**Issue by DarthJahus** [wekan/wekan#5351]
### Server Setup Information

* Did you test in newest Wekan? Yes
* Did you configure root-url correctly so Wekan cards open correctly (see https://github.com/wekan/wekan/wiki/Settings)? Yes
* Operating System: Ubuntu 23.04 Server
* Deployment Method (Snap/Docker/Sandstorm/bundle/source): Docker
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): Apache
* Node.js Version: ?
* MongoDB Version: 6
* What webbrowser version are you using (Wekan should work on all modern browsers that support Javascript)? Latest MS Edge

#### Reproduction Steps

Activated "Automatically add users with the domain name" on Organisation. However, users that have that domain name in their email address are not automatically added to the organisation upon sign up.

**Comment by DarthJahus** [comment]
Can it be related to this? https://github.com/wekan/wekan/issues/1426

**Comment by mivalsten** [comment]
I can confirm that it does not work properly on version 7.60.0 from docker.

**Comment by xet7** [comment]
Yes, developing this feature is still in progress. It is not ready yet.
